pandas/core/internals.py

```diff
@@ -5313,6 +5330,10 @@ def concatenate_block_managers(mgrs_indexers, axes, concat_axis, copy):
     return BlockManager(blocks, axes)
 
 
+def is_sparse_join_units(join_units):
+    return all(type(ju.block) is SparseBlock for ju in join_units)
```

Comment:
This is very similar (and maybe could be DRYed up) like pd.concat([sparse, sparse])